test/bulk.test.ts

```diff
@@ -6,11 +6,41 @@ import ConnectionManager from './helper/connection-manager';
 import config from './config';
 import { isObject, isString } from './util';
 import { isNodeJS } from './helper/env';
+import { BulkIngestBatchResult } from 'jsforce/lib/api/bulk';
 
 const connMgr = new ConnectionManager(config);
 const conn = connMgr.createConnection();
 conn.bulk.pollTimeout = 90 * 1000; // adjust poll timeout to test timeout.
 
+export async function insertAccounts(
+  id: string | number,
+  qty: number = 20,
+): Promise<BulkIngestBatchResult> {
+  if (typeof id === 'number') {
+    id = id.toString();
+  }
+
+  const insertRecords = [
+    ...Array.from(Array(qty), (a, i) => ({
+      Name: `Bulk Account ${id} #${i + 1}`,
```

Comment:
put a unique id in the `Name` field, tests can use it later to query specifically their account records.